src/masonite/cache/drivers/FileDriver.py

```diff
@@ -0,0 +1,104 @@
+import os
+from ...utils.filesystem import make_full_directory, modified_date
+from pathlib import Path
+import pendulum
+import json
+import glob
+
+
+class FileDriver:
+    def __init__(self, application):
+        self.application = application
+
+    def set_options(self, options):
+        self.options = options
+        if options.get("location"):
+            make_full_directory(options.get("location"))
+        return self
+
+    def add(self, key, value):
+        if self.has(key):
+            return self.get(key)
+
+        self.put(key, value)
+        return self.get(key)
+
+    def get(self, key, default=None, **options):
+        if not self.has(key):
+            return None
+
+        modified_at = self.get_modified_at(os.path.join(self._get_directory(), key))
+
+        with open(os.path.join(self._get_directory(), key), "r") as f:
+            value = f.read()
+
+            if modified_at.add(seconds=self.get_cache_expiration(value)).is_past():
+                self.forget(key)
+                return default
+
+            value = self.get_value(value)
+
+        return value
+
+    def put(self, key, value, seconds=None, **options):
+
+        time = self.get_expiration_time(seconds)
+
+        if isinstance(value, (dict,)):
+            value = json.dumps(value)
+
+        with open(os.path.join(self._get_directory(), key), "w") as f:
+            f.write(f"{time}:{value}")
+
+    def has(self, key):
+        return Path(os.path.join(self._get_directory(), key)).exists()
+
+    def increment(self, key, amount=1):
+        return self.put(key, str(int(self.get(key)) + amount))
+
+    def decrement(self, key, amount=1):
+        return self.put(key, str(int(self.get(key)) - amount))
+
+    def remember(self, key, callable):
+        value = self.get(key)
+
+        if value:
+            return value
+
+        callable(self)
+
+    def forget(self, key):
+        try:
+            os.remove(os.path.join(self._get_directory(), key))
+            return True
+        except FileNotFoundError:
+            return False
+
+    def flush(self):
+        files = glob.glob(f"{self._get_directory()}/*")
+        for f in files:
+            os.remove(f)
+
+    def _get_directory(self):
+        return self.options.get("location")
+
+    def get_modified_at(self, filename):
+        return pendulum.from_timestamp(modified_date(filename))
+
+    def get_expiration_time(self, seconds):
+        if seconds is None:
+            seconds = 9999999
```

Comment:
this is the max expiration time ? 
For HTTP cache there are no conventions but we often see something like 1 year so (31557600 s)
Here the value is a bit less than 4 months, isn't it a bit short ? 

Or maybe this value should go in the config with a default of one year ?